sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -318,6 +318,8 @@ def _wrapper_is_flat_memory(physical_port):
         try:
             sfp = platform_chassis.get_sfp(physical_port)
             api = sfp.get_xcvr_api()
+            if not api:
```

Comment:
could it cover some issues? Will get_xcvr_api somehow return None due to some problem?

Reply:
Yes, it can happen for RJ45 cables

Reply:
Is RJ45 cable the only case that sfp.get_xcvr_api() will return None on our platform?